src/tools/clippy/tests/ui/deprecated.rs

```diff
@@ -18,5 +18,6 @@
 #![warn(clippy::filter_map)]
 #![warn(clippy::pub_enum_variant_names)]
 #![warn(clippy::wrong_pub_self_convention)]
+#![warn(clippy::forget_copy)]
```

Comment:
Looks like this got left behind, it can be removed

Reply:
Yeah, left over from the `cargo dev deprecate`. Should now be fixed.